py4DSTEM/io/utils.py

```diff
@@ -0,0 +1,37 @@
+from os.path import splitext
+
+def parse_filetype(fp):
```

Comment:
might be worth doing string santizing e.g `fext.lower()` incase someone does `* .dM3`

Reply:
ok